op-chain-ops/upgrades/l1.go

```diff
@@ -122,7 +117,7 @@ func L1CrossDomainMessenger(batch *safe.Batch, implementations superchain.Implem
 		return fmt.Errorf("OtherMessenger address doesn't match config")
 	}
 
-	calldata, err := l1CrossDomainMessengerABI.Pack("initialize", superchainConfigProxy, optimismPortal)
+	calldata, err := l1CrossDomainMessengerABI.Pack("initialize", common.HexToAddress(superchainConfig.Config.SuperchainConfigAddr.String()), optimismPortal)
```

Comment:
`common.Address(superchainConfig.Config.SuperchainConfigAddr)` should work. It copies when you pass it as argument anyway. And the cast is ok because the underlying type is both `[20]byte`